privacy-protections/click-to-load/index.html

```diff
@@ -6,33 +6,35 @@
+<div class="fb-like" data-href="https://privacy-test-pages.glitch.me/" data-width="" data-layout="standard" data-action="like" data-size="small" data-share="true"></div>
+<br><div class="fb-like" data-href="https://privacy-test-pages.glitch.me/" data-width="" data-layout="standard" data-action="recommend" data-size="small" data-share="true"></div>
+<br><div class="fb-save" data-uri="https://privacy-test-pages.glitch.me/" data-size="small"><p>Save button (appears to be broken in FB docs)</p></div>
+<br><div class="fb-share-button" data-href="https://privacy-test-pages.glitch.me/" data-layout="button_count" data-size="small"><a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins%2F&amp;src=sdkpreparse" class="fb-xfbml-parse-ignore">Share</a></div>
 
 <h2> All the social buttons in iFrames </h2>
-<iframe src="https://www.facebook.com/plugins/like.php?href=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins2%2F&width=450&layout=standard&action=like&size=small&share=true&height=35&appId" width="450" height="35" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"></iframe>
-<br /><iframe src="https://www.facebook.com/v2.0/plugins/like.php?href=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins1%2F&width=450&layout=standard&action=like&size=small&share=true&height=35&appId" width="450" height="35" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"></iframe>
-<br /><iframe src="https://www.facebook.com/plugins/share_button.php?href=https%3A%2F%2Fdevelopers.facebook.com%2Fdocs%2Fplugins%2F&layout=button_count&size=small&width=121&height=20&appId" width="121" height="20" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"></iframe>
+<iframe src="https://www.facebook.com/plugins/like.php?href=https%3A%2F%2Fprivacy-test-pages.glitch.me%2F&width=450&layout=standard&action=like&size=small&share=true&height=35&appId" width="450" height="35" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"></iframe>
+<br><iframe src="https://www.facebook.com/plugins/like.php?href=https%3A%2F%2Fprivacy-test-pages.glitch.me%2F&width=450&layout=standard&action=recommend&size=small&share=true&height=35&appId" width="450" height="35" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"></iframe>
+<p> SAVE button N/A in iframes per FB docs </p>
```

Comment:
I don't understand this note about save buttons. Perhaps you could explain a bit more and link to the relevant part of the Facebook docs for context?

Reply:
Iframes aren't supported for save button (which is being deprecated and I couldn't get to work anyway).  I just deleted this comment